perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -593,13 +593,23 @@ sub find_closest {
     my $lon = $problem ? $problem->longitude : $data->{longitude};
     my $j = $problem ? $problem->geocode : undef;
 
-    if (!$j) {
-        $j = FixMyStreet::Geocode::reverse($self, $lat, $lon);
-        if ($problem) {
-            # cache the geocoder results for use in alerts
-            $problem->geocode( $j );
-            $problem->update;
-        }
+    if ($j) {
+        my $addr = FixMyStreet::Geocode::Address->new($j);
+
+        # Cobrand's geocoder may have changed; only return if it hasn't
+        return $addr if ref($addr) =~ $self->get_reverse_geocoder;
```

Comment:
I don't think we want to make this change, because this unnecessarily stops all previous geocodes working if a cobrand changes its provider. The purpose of the Geocode::Address wrapper objects is that it shouldn't matter which provider is used in general.
If we care about old reports, we could in theory do a manual update of their entries, but I don't think that's necessary here.